### PR TITLE
F/hipchat docs

### PR DESCRIPTION
**Pull request by jugarrit**
This won't be merged until hipchat is ready (of course)

### CODE REVIEW

**Comment by dannytranlx** [comment]
Can you redo this screenshot or crop the tooltip out :joy: 

asfd

**Comment by dannytranlx** [comment]
:maple_leaf:

File: source/includes/integrations/_hipchat.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+Hipchat is a team-based chat tool that helps to group all of your team's conversations and workflow into one beautiful, searchable place.
+
+Smooch has a Hipchat integration that allows your team to communicate with your app's users, without leaving Hipchat.
+
+### Configuring Hipchat
+
+#### Step 1. Authorize Smooch so that it can talk to your Hipchat
+
+<br/>
+<img style="width:50%; min-width:400px; max-width:800px;" src="/images/hipchat_smooch_integration.png" alt="Hipchat integration">
```
Comment (dannytranlx) [inline source/includes/integrations/_hipchat.md]: I added this class that should be doing the same thing!
`.half-width-img`

File: source/includes/integrations/_hipchat.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+## Hipchat
```
Comment (dannytranlx) [inline source/includes/integrations/_hipchat.md]: All `Hipchat` copy should be `HipChat`

image

**Comment by dannytranlx** [comment]
:maple_leaf:

File: source/includes/integrations/_hipchat.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+### Configuring Hipchat
+
+#### Step 1. Authorize Smooch so that it can talk to your Hipchat
+
+<br/>
+<img style="width:50%; min-width:400px; max-width:800px;" src="/images/hipchat_smooch_integration.png" alt="Hipchat integration">
+
+#### Step 2. ...
+
+![There is no Step 2](/images/step_2.jpeg)
```
Comment (alavers) [inline source/includes/integrations/_hipchat.md]: Use imgur.com/memegen for better great success

http://i.imgur.com/TEYe7WC.png

Comment (juliangarritano) [inline source/includes/integrations/_hipchat.md]: Nice!

**Comment by dannytranlx** [comment]
:maple_leaf:

**Comment by mspensieri** [comment]
:+1:

**Comment by alavers** [comment]
:+1:

**Pull request merged**